doc/README.md

```diff
@@ -0,0 +1,25 @@
+# Automated markdown manpage
+
+Make a markdown document with the usage messages of (selected) `vg` subcommands. 
+Calls the `vg` command, so it will match the version available in the command line.
+
+```sh
+python3 vgmanmd.py > man.md
+```
+
+Then copy the markdown content to a wiki page, for example, or move the markdown page to somewhere in the vg repo?
```

Comment:
There's a checkout of the wiki as a submodule, so this maybe should involve writing to `wiki/vg-manpage.md` and committing and pushing.

This could also become a Makefile target e.g. `make manpage`.